### DISCUSSION TITLE
Server 56 UK / INT

### DISCUSSION THREAD

**Discussion by Zece11** [DinoDevs/GladiatusCrazyAddon#401] [Q&A] [answered]
Hello Dino,

as you know or not the new server 56 uk / INT was opened, where can we find it for arena or for expedition simulator ? 
It will be added one day or ? :D 

Thank you for all your work.

Gladiatus Comunity

**Comment by GramThanos** [comment, marked as answer]
> the new server 56 uk / INT was opened, where can we find it for arena or for expedition simulator ? 

Currently our automated scripts generates the gladiatus server list based on the servers listed under the country (e.g. https://en.gladiatus.gameforge.com/game/index.php?mod=start&submod=highscore ) and as you can see, this server is missing from that list (I don't know why). Thus we will have to manually fix that, which may take some time.

**Reply by Zece11** [reply]
as a new server would help us a lot evolving if we had at least the expedition simulator :D :D

**Reply by GramThanos** [reply]
clear browser cache and try again at the simulator's  page

**Reply by Zece11** [reply]
working now , i love you man <3
